Scarb.toml

```diff
@@ -10,10 +10,10 @@ license-file = "LICENSE"
 keywords = ["fixed-point", "wad", "ray", "cairo", "starknet"]
 
 [dependencies]
-starknet = "2.3.1"
+starknet = "2.4.0"
```

Comment:
Just occurred to me this is a general purpose lib. It does not have to have `starknet` as a dependency, because it can be used outside of the context of Starknet.

I haven't verified it, but I guess we can remove the whole `[dependencies]` section and `starknet` should be pulled in as a dev dependency via `snforge_std`.